internal/controller/ols_app_server_assets.go

```diff
@@ -93,10 +93,18 @@ func (r *OLSConfigReconciler) generateOLSConfigMap(cr *olsv1alpha1.OLSConfig) (*
 			LibLogLevel: cr.Spec.OLSConfig.LogLevel,
 		},
 		ConversationCache: conversationCache,
+		TLSConfig: TLSConfig{
+			TLSCertificatePath: path.Join(OLSAppCertsMountRoot, OLSCertsSecretName, "tls.crt"),
+			TLSKeyPath:         path.Join(OLSAppCertsMountRoot, OLSCertsSecretName, "tls.key"),
+		},
 	}
 
 	devConfig := DevConfig{
-		DisableTLS: true,
+		DisableTLS: cr.Spec.OLSConfig.DisableTLS,
```

Comment:
let's add an option to disable authentication, too. We have not yet figured out how to pass an authentication header from the console plugin. 
https://github.com/openshift/lightspeed-service/blob/main/examples/olsconfig.yaml#L68

Reply:
I am updating this PR  @raptorsun